diff --git a/shared/f7.js b/shared/f7.js
index eea0d3d58c9adefea8cbc2ca0c489aa349e45ed9..56e7d33ca139840543f8604bb44c07666f943626 100644
--- a/shared/f7.js
+++ b/shared/f7.js
@@ -2,7 +2,7 @@
 import Framework7 from 'framework7/lite';
 import { extend, unsetRouterIds } from './utils.js';
 let f7;
-let f7events;
+let f7events = new Framework7.Events();
 const theme = {};
 const f7routers = {
   views: [],
